### DISCUSSION TITLE
10.1 New swipe animation slower?

### DISCUSSION THREAD

**Discussion by chris42** [ReadYouApp/ReadYou#754] [General]
Just got the update to 10.1, so thank you for your work and effort you put into this app!

Is there a way to speed up the swipe to mark read animation, like in the previous versions?
In this version the marking fades away, however slower than I read or decide on the news item. Hence when swiping away a longer list I am waiting a lot on the app to finish the animation.

**Comment by JunkFood02** [comment]
If you use "mark article as read" heavily, I'll suggest you long press on an item and use "Mark above/below as read"

**Reply by chris42** [reply]
How do I mark the ones that I want to keep with that?

**Reply by JunkFood02** [reply]
Star the items that you want to read later

**Reply by chris42** [reply]
So the proposal is to use two different sorting mechanisms, that partly need multiple touches.

Maybe to get back to my question:
Why was the animation changed in the first place? Why does it need to be slower?
